lib/commands/release.js

```diff
@@ -300,22 +215,151 @@ module.exports = {
+          description: "the name of the git tag to create"
+        },
+        {
+          name: 'annotation',
+          type: String,
+          aliases: [ 'a' ],
+          description: "a message passed to the `--message` option of `git tag`, indicating that to the tag should be created with the `--annotated` option (defualt is lightweight), the string '%@' is replaced with the tag name"
+        },
+        {
+          name: 'message',
+          type: String,
+          aliases: [ 'm' ],
+          default: 'Released %@',
+          description: "a message passed to the `--message` option of `git commit`, the string '%@' is replaced with the tag name"
+        },
+        {
+          name: 'manifest',
+          type: Array,
+          default: [ 'package.json', 'bower.json' ],
+          description: "a set of JSON files to replace the top-level `version` property in with the new tag name"
+        },
+        // {
+        //   name: 'build',
+        //   type: Boolean,
+        //   aliases: [ 'b' ],
+        //   default: false,
+        //   description: "whether to run `ember build` or not"
+        // },
+        // {
+        //   name: 'publish',
+        //   type: Boolean,
+        //   aliases: [ 'p' ],
+        //   default: false,
+        //   description: "whether to run `npm publish` post-tagging or not"
+        // },
+        {
+          name: 'yes',
+          type: Boolean,
+          aliases: [ 'y' ],
+          default: false,
+          description: "whether to skip confirmation prompts or not (answer 'yes' to all questions)"
+        },
+        {
+          name: 'strategy',
+          type: String,
+          aliases: [ 's' ],
+          default: 'semver',
+          description: "strategy for auto-generating the tag name, either 'semver' or 'date', ignored if the 'name' option is specified"
+        },
+        {
+          name: 'major',
+          type: Boolean,
+          aliases: [ 'j' ],
+          description: "when strategy is 'semver', specifies that the major version number should be incremented"
+        },
+        {
+          name: 'minor',
+          type: Boolean,
+          aliases: [ 'i' ],
+          description: "when strategy is 'semver', specifies that the minor version number should be incremented, ignored if '--major' option is true"
+        },
+        {
+          name: 'format',
+          type: String,
+          aliases: [ 'f' ],
+          default: 'YYYY.MM.DD',
+          description: "when strategy is 'date', the format used to generate the tag"
+        },
+        {
+          name: 'timezone',
+          type: String,
+          aliases: [ 'z' ],
+          default: 'UTC',
+          description: "when strategy is 'date', the timezone to consider the current date in"
+        }
+      ].map(function(option){
+        if ((optionsFromConfig[option.name] !== undefined) && (option.default !== undefined)) {
+          option.default = optionsFromConfig[option.name];
```

Comment:
Ah, yes, forgot that part of our conversation, currently I believe you would have some cryptic error message. I won't have time to work on this for a bit, but this would be a nice improvement.